certification/engine/engine.go

```diff
@@ -97,6 +97,7 @@ var hasNoProhibitedCheck certification.Check = &shell.HasNoProhibitedPackagesChe
 var validateOperatorBundle certification.Check = &shell.ValidateOperatorBundleCheck{}
 var scorecardBasicSpecCheck certification.Check = &shell.ScorecardBasicSpecCheck{}
 var scorecardOlmSuiteCheck certification.Check = &shell.ScorecardOlmSuiteCheck{}
+var relatedImageManifestSchemaVersionCheck certification.Check = &shell.RelatedImagesAreSchemaVersion2Check{}
```

Comment:
How about `csvImageSchemaVersionCheck`? The check name is a bit ambiguous.

Reply:
The CSV contains many locations where an image may be referenced (e.g. related images vs. deployment images).